packages/engine-ui/src/components/forms/inputs/DateInput.tsx

```diff
@@ -21,10 +21,19 @@ const TimeRestrictionDateInput = React.forwardRef<
     placeholder: string
     withIcon: boolean
     handleFocus?: () => void
+    shouldBeRequired: boolean
   }
 >(
   (
-    { onChange, onClick, value, placeholder, withIcon = true, handleFocus },
+    {
+      onChange,
+      onClick,
+      value,
+      placeholder,
+      withIcon = true,
+      handleFocus,
+      shouldBeRequired = true,
```

Comment:
Do we want default to be required? Think opposite is more common

Reply:
Also, naming, `should` is kind of vague, think just `required` would do it, a bit more straightforward?

Reply:
Doesn't work to name it required, gets lost some way in the prop/forwardref.. 🤷‍♂️

Reply:
Then, isRequired?